extensions/interactions/AlgebraicExpressionInput/directives/oppia-interactive-algebraic-expression-input.component.ts

```diff
@@ -135,6 +140,10 @@ export class AlgebraicExpressionInputInteractionComponent implements OnInit {
       submitAnswer, isCurrentAnswerValid);
   }
 
+  ngOnDestroy(): void {
+    this.viewIsDestroyed = true;
```

Comment:
I searched for a method dedicated to unbinding an event handler in the Guppy repo, but was unable to find one.

An alternative is to set a flag, and use it to determine when detectChanges() should be called.
`viewIsDestroyed` is initialized to be false, and is set to true in the onDestroy method, which prevents any future detectChanges() calls.